books/free-programming-books-langs.md

```diff
@@ -108,7 +108,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
     * [Ionic](#ionic)
     * [jQuery](#jquery)
     * [meteor](#meteor)
-    * [Next.js](#next.js)
+    * [Next.js](#nextjs)
```

Comment:
I think you fail with the right branch committing 0b5884874b69a0a23f0f5bccacf6775cce0d3e0b lol. Maybe this goes in the other PR?

Could you cherry pick this 0b5884874b69a0a23f0f5bccacf6775cce0d3e0b into #7020 (https://github.com/EbookFoundation/free-programming-books/tree/fix-caps-in-frags) and revert/hard reset this change here?

Reply:
Sorry, I missunderstand the PR attending to its title. I update it